src/datomic_toolbox/core.clj

```diff
@@ -1,20 +1,28 @@
 (ns datomic-toolbox.core
   (:require [clojure.java.io :as io]
             [clojure.walk :as walk]
-            [datomic.api :as d]
-            [turbovote.resource-config :refer [config]])
+            [datomic.api :as d])
   (:refer-clojure :exclude [partition]))
 
+(def default-uri (atom nil))
+(def default-partition (atom nil))
+
+(defn configure! [{:keys [uri partition]}]
+  (reset! default-uri uri)
+  (reset! default-partition partition))
+
+(defn uri [] @default-uri)
+
 (defn connection []
-  (d/connect (config [:datomic :uri])))
+  (d/connect (uri)))
+
+(defn partition [] @default-partition)
 
 (def db (comp d/db connection))
```

Comment:
Could we make this a `defn` while we're in here? (I know it's not something you're touching, so it's not a blocker.)

`defn` sets up nice metadata that contains the function signature. Using `comp` makes it look like there might be other arguments to pass it.